test/utils/generate-tokenization-tests.ts

```diff
@@ -194,11 +217,13 @@ function loadTests(dataDirPath: string): LoadedTest[] {
                     setName,
                     name: descr.description,
                     input: descr.input,
-                    expected: concatCharacterTokens(expected),
+                    expected,
                     initialState: getTokenizerSuitableStateName(initialStateName),
                     initialStateName,
                     lastStartTag: descr.lastStartTag,
-                    expectedErrors: descr.errors || [],
+                    expectedErrors: TestsWithBrokenErrors[descr.description]
```

Comment:
Broken tests are now referenced here, instead of when comparing once all errors are gathered.